py/kubeflow/testing/cleanup_ci.py

```diff
@@ -611,7 +787,11 @@ def cleanup_all(args):
          cleanup_service_account_bindings,
          cleanup_workflows,
          cleanup_disks,
+         cleanup_forwarding_rules,
+         cleanup_target_http_proxies,
+         cleanup_url_maps,
          cleanup_backend_services,
+         cleanup_instance_groups,
```

Comment:
I think this one needs to be after `cleanup_health_checks`.

Reply:
Done.